### PR TITLE
GraphQL

### PR DESCRIPTION
**Pull request by hizmailovich**
## PR-Codex overview
This PR adds GraphQL functionality to the movie microservice, including new queries and mutations, a schema, and GraphQL error handling. It also updates dependencies and removes unused code.

### Detailed summary
- Adds GraphQL functionality including schema, queries, and mutations
- Adds GraphQL error handling
- Updates dependencies
- Removes unused code

> The following files were skipped due to too many changes: `src/main/java/com/solvd/movie/web/controller/MovieController.java`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: codecov.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+coverage:
+  status:
+    patch: off
```
Comment (h1alexbel) [inline codecov.yml:3]: @hizmailovich don't do it, bad idea

**Review by h1alexbel — CHANGES_REQUESTED** [review]

File: .github/workflows/codecov.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@ jobs:
       - uses: actions/setup-java@v3
         with:
           distribution: 'temurin'
-          java-version: 17
+          java-version: 19
```
Comment (h1alexbel) [inline .github/workflows/codecov.yml:15]: @hizmailovich these changes should be located in the separate pr

Comment (hizmailovich) [reply .github/workflows/codecov.yml]: Updated java-version from pom.xml, works fine with java 17

**Comment by hizmailovich** [comment]
@h1alexbel Can I merge this pr?

**Review by h1alexbel — APPROVED** [review]
@hizmailovich lgtm, but we can't merge this one, until ci gate says OK

**Comment by hizmailovich** [comment]
@rultor merge

**Pull request merged**